### ISSUE TITLE
VPR warning verbosity with timing analysis on and custom routing graphs with isolated clock routing

### ISSUE THREAD

**Issue by petergrossmann21** [verilog-to-routing/vtr-verilog-to-routing#2665]
I am modeling an architecture with dedicated clock IOBs and a custom routing graph that isolates the clock routing from general purpose routing channels.  The isolation of the clock routing results in a disjoint routing graph.

If I perform place and route with this model and timing analysis turned on, the function

`timing_driven_route_connection_common_setup`

trips a large (thousands) number of warnings of the form

`Warning 1185: No routing path for connection to sink_rr 175371, leaving unrouted to retry later`

There are no equivalent warnings when timing analysis is turned off.  In either case, place and route for a large variety of benchmarks is successful, and timing reports are generated as expected.

As the goal is to offer such an architecture with an open source CAD flow to customers, elimination of spurious warnings is important.  

After some initial investigation, it appears that VPR is attempting to estimate the delay from every OPIN to every IPIN, and since my routing graph is disjoint it is not able to reach a solution for many OPIN/IPIN pairs.  If my understanding is correct the warning is reasonable when the assumption is made that all RR graphs should have valid routes from every startpoint to every endpoint, but in the case where this assumption is "legally" violated VPR's messaging may no longer make sense.  Specifically, if an architect wishes to only permit a subset of IOBs to have clocks placed on them, it would appear necessary to constrain this setup function so that when clock startpoints can only route to clock endpoints, there are no attempts to check their routes/delays to non-clock endpoints.

Given that I am describing corner case behavior, I am reluctant to propose too invasive of changes or request much of anyone's time, and would appreciate brief suggestions on how to resolve the high warning count safely.  This topic also closely relates to @kimiatkh 's work from last year:  https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2386 https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2446

**Comment by vaughnbetz** [comment]
@petergrossmann21 : I agree these warnings should not be issued without a higher verbosity level being set, as this is a valid architecture.  This warnings is being printed when we expand the bounding box during routing as the original bounding box around the net terminals didn't yield a successful routing. This certainly could happen for clock nets where we are routing with a big H-tree, and some of the drive points might be outside the net bounding box.

Can you check if these warnings are still there (from looking at the code I think they will be) and if they are, change the code to
VTR_LOGV (route_verbosity > 1, "usual message here")

Then routing CAD developers can still turn this on, but no one else will see it.

**Comment by vaughnbetz** [comment]
@petergrossmann21 : I think we fixed this ... can you confirm it is fixed so I we can close this?

**Comment by petergrossmann21** [comment]
@vaughnbetz Confirmed and closed.
